contracts/src/spaces/facets/membership/MembershipBase.sol

```diff
@@ -30,6 +29,7 @@ abstract contract MembershipBase is IMembershipBase {
 
     if (info.freeAllocation > 0) {
       _verifyFreeAllocation(info.freeAllocation);
+      ds.freeAllocationEnabled = true;
```

Comment:
`freeAllocationEnabled` seems redundant. Where do we need it instead of `freeAllocation > 0`?

Reply:
in _getMembershipFreeAllocation, it's a requirement for default memberhsip limit via platform requirements to work and for prepay to work when charging